src/activate/handleTask.ts

```diff
@@ -17,9 +15,9 @@ export const handleNewTask = async (params: { prompt?: string } | null | undefin
 	}
 
 	if (!prompt) {
-		await vscode.commands.executeCommand("roo-cline.SidebarProvider.focus")
+		await vscode.commands.executeCommand(`${Package.name}.SidebarProvider.focus`)
```

Comment:
As a follow-up we might want to add a `vscode` utility that packages up wraps command execution for better DX.

For example:
```typscript
export const focusSidebar = () =>
  vscode.commands.executeCommand(`${Package.name}.SidebarProvider.focus`)
```